### DISCUSSION TITLE
I search  wire-dirty property for  form to save data in db

### DISCUSSION THREAD

**Discussion by sergeynilov** [livewire/livewire#8062] [1. Help]
Reading how works wire-dirty property at 
https://livewire.laravel.com/docs/wire-dirty
 I check  and see it is not synced to database when user saves the data.
 
 If there is a way in   laravel 10/livewire 3 to make similar synced when user submits the form to save data in db ?

**Comment by joshhanley** [comment]
@sergeynilov I'm not sure exactly what you mean but I think Laravel models have an `isDirty()` method or something like that, which you might be able to use.

**Reply by sergeynilov** [reply]
I did not fond any isDirty mentioned here https://livewire.laravel.com/docs/  or searchin in net. Please reference if there is isDirty method

**Reply by joshhanley** [reply]
@sergeynilov I said Laravel, so try the Laravel docs as it will be a method on a Laravel model 🙂

**Reply by sergeynilov** [reply]
Yes, I found this https://stackoverflow.com/questions/28836013/what-does-isdirty-mean-in-laravel branch, but it seems tricky. How can i implement it ?

**Reply by joshhanley** [reply]
@sergeynilov how to use it is documented in the Laravel docs, I suggest using them before going to stackoverflow https://laravel.com/docs/10.x/eloquent#examining-attribute-changes
